Model/Method/Ideal.php

```diff
@@ -150,11 +143,6 @@ public function validate()
 
         $paymentInfo = $this->getInfoInstance();
 
```

Comment:
Removing this part of code it will make the issue BP-1257 Ideal payment method selection triggers "Please select a issuer from the list" error to be triggered again, we should to find a way to skip validation on set payment information

Reply:
@LucianTuriacArnia maybe it's an option to have a default issuer as fallback? And make it a configuration within the module.

Reply:
@rbnmulder, that's a valid point. While utilizing defaults could be a potential solution, we are exploring alternative approaches to resolve this without relying on defaults.